ninja-core/src/main/java/ninja/template/TemplateEngineFreemarkerExceptionHandler.java

```diff
@@ -53,38 +52,9 @@ public TemplateEngineFreemarkerExceptionHandler(Logger logger,
 
     public void handleTemplateException(TemplateException te,
```

Comment:
So like a dev version of the general error page which does a message+stack print? That would be pretty cool, I'll play around with that idea and open another PR focus on that idea.